app/workers/tipline_statistics_periodic_check_worker.rb

```diff
@@ -4,10 +4,8 @@ class TiplineStatisticsPeriodicCheckWorker
   sidekiq_options retry: 0
 
   def perform
-    if CheckConfig.get('ENABLE_TIPLINE_STATS_MONITORING', true)
-      periodic_check = Time.now.ago(1.day)
-      count = MonthlyTeamStatistic.where('created_at > ? OR updated_at > ?', periodic_check, periodic_check).count
-      CheckSentry.notify(StandardError.new('No MonthlyTeamStatistic updated or created in the last 24 hours')) if count == 0
-    end
+    periodic_check = Time.now.ago(26.hours)
```

Comment:
I think even 25 would be good, but I suggest you add a comment here just to make clear that it was not a typo, but an intended buffer (something like "adding a buffer of one hour to our expected 24 hours interval", something like that.

Reply:
Done